Makefile

```diff
@@ -224,3 +224,7 @@ kubelet:
 .PHONY: kube-proxy
 kube-proxy:
 	KUBE_GIT_VERSION=$(KUBE-PROXY_GIT_VERSION) KUBE_BUILD_PLATFORMS=windows/amd64 make -C kube-proxy WHAT=cmd/kube-proxy
+
+.PHONY : containerd
+containerd:
+	GOOS=windows VERSION=$(shell cd containerd && git describe --match 'v[0-9]*' --dirty='.m' --always) make -C containerd WHAT=bin/containerd.exe
```

Comment:
Do we need to set the [REVISION](https://github.com/containerd/containerd/blob/main/Makefile#L34) as well?

Reply:
It gets set from the containerd Makefile, since it is not getting pre-set in our Dockerfile before the containerd make target runs we are good.
PTAL at the [output comment](https://github.com/openshift/windows-machine-config-operator/pull/1423#issuecomment-1437820355), revision comes after the hyphen.